operations/windows-cell.yml

```diff
@@ -51,6 +52,11 @@
               ca_cert: "((diego_bbs_client.ca))"
               client_cert: "((diego_bbs_client.certificate))"
               client_key: "((diego_bbs_client.private_key))"
+            consul:
+              ca_cert: "((consul_agent.ca))"
```

Comment:
Hey @jaresty. This is a bit confusing, but the `ca_cert` here is used by the rep to determine if it's talking to a trusted agent. The agent is running with the `consul_agent.ca`, not the `diego_consul_client.ca` (which do happen to be the same in this case.)

The rep then presents its certificate to the agent for mutual TLS. The agent will verify that the `diego_consul_client.certificate` was signed by the `diego_consul_client.ca`, which happens to be the same as `consul_agent.ca`. So if you really wanted us to, we would change [this line](https://github.com/cloudfoundry/cf-deployment/pull/41/files/be40a85443c4e6f76bf718eb5725f4145eb3a23a#diff-f73030c6ed757d99f2745c7d5d85d186R36) to include both the `diego_consul_client.ca` and `consul_agent.ca`, but they'd be identical and I don't know what that would do.